src/model/neural_network/neural_network_adapter.py

```diff
@@ -33,6 +38,8 @@ def get_instructions(self, distances: List[float], speed: float) -> CarInstructi
                 f"Neural network input neurons: {self.neural_network.input_layer_neuron_count}.\n"
                 f"Sensors count: {len(distances)}."
             )
-        valid_input = np.expand_dims(np.array(distances + [speed]), 0)
+        valid_input = np.expand_dims(
+            np.array(distances + [speed]) * self._PREPROCESSING_RATE, 0
```

Comment:
Use `NeuralNetworkAdapter._PREPROCESSING_RATE`

Reply:
What's the difference?